### ISSUE TITLE
SNR and noise level

### ISSUE THREAD

**Issue by jukeeva** [leoliuf/MRiLab#51]
Hi, I was wondering how is the noise level measured? Is the value in db or is it the(SNR) ratio of the average signal intensity over the standard deviation of the noise? The formula in the manual is hard to follow and it would be nice to have some kind of measurement unit also. Thank you.

**Comment by DreymaR** [comment]
SNR is the ratio of signal in a given region (S) to noise in that region, which is commonly taken to be the signal deviation σ of the signal (in a non-zero signal region). It's really a bit more complex, but that's the gist of it.

Since it is a ratio of signals, it has no unit.

**Comment by jukeeva** [comment]
ah thank you! so it is just background noise. Because there is this formula from user guide and it seems like Noise level and SNR are not the same thing, so, I got confused 
![image](https://github.com/leoliuf/MRiLab/assets/44908342/6d0c1adc-b389-43fb-bfa7-d877ee846833)

**Comment by jukeeva** [comment]
ah thank you! so it is just background noise. Because there is this formula from user guide and it seems like Noise level and SNR are not the same thing, so, I got confused 
![image](https://github.com/leoliuf/MRiLab/assets/44908342/6d0c1adc-b389-43fb-bfa7-d877ee846833)

**Comment by DreymaR** [comment]
They are indeed not the same thing. As I said, SNR is the ratio of signal S to Noise N in your voxel or whatever.

The formula contains factors that affect this ratio: Some affect S, some N and some both. B0, for instance, increases S, as does the square root of NEX. Bandwidth increases N by its square root. Etc etc.

**Comment by jukeeva** [comment]
ok, thank you!
